modules/tensor_mechanics/include/actions/TensorMechanicsAction.h

```diff
@@ -75,9 +75,9 @@ class TensorMechanicsAction : public TensorMechanicsActionBase
   enum class PlanarFormulation
   {
     None,
+    PlaneStress,
```

Comment:
PlaneStress might be misleading here, because this is a weak enforcement. Can it be WeakPlaneStress?

Reply:
Yeah, good point. I don't know if we would ever want to offer a traditional plane stress implementation -- the WeakPlaneStress approach is the way we're going with for now. Do you think that the user should have to say `planar_formulation = WEAK_PLANE_STRESS` in the input file? It might help indicate that it's not a standard plane stress implementation.

Reply:
@hugary1995 We discussed this amongst ourselves, and we agree that using WEAK_PLANE_STRESS is better because it's more descriptive, and it leaves the PLANE_STRESS option available for use in the future in a traditional plane stress implementation. I just pushed an updated version that makes that change.